lib/cachex/services/courier.ex

```diff
@@ -80,7 +80,8 @@ defmodule Cachex.Services.Courier do
             normalized = normalize_commit(result)
 
             with { :commit, val } <- normalized do
-              Put.execute(cache, key, val, const(:notify_false))
+              options = Keyword.merge(options, const(:notify_false))
```

Comment:
Annotation: We are still merging the `const(:notify_false)` on top of whatever options are provided, ensuring that the `const(:notify_false)` will still take effect.